### ISSUE TITLE
Allow filter by Not-Worked (needed) band on Awards window

### ISSUE THREAD

**Issue by vazquezrodjoel** [foldynl/QLog#510]
It would be helpful if we could filter the needed bands on the award window.  Like in below, there are some states I need on some bands, but while I can select the "Not-Worked Only" filter, it will skip those bands since I have worked that entity before, but only on some bands.
![image](https://github.com/user-attachments/assets/9699162d-e74a-4a6f-900f-dee0ec1a42d9)

With the "Not-Worked Only" filter:
![image](https://github.com/user-attachments/assets/efcef98a-e926-492a-afab-9c7c525aa55b)

**Comment by aa5sh** [comment]
So if I understand you want to a selection for all bands or certain bands.  So you could pick 20,40,80 for instance and show which you haven't worked on those three bands?  Something like that?

**Comment by vazquezrodjoel** [comment]
yes, for example in the second image if you notice lets say I want to focus to complete the WAS on 30m, 17m or 15m, it would be nice to be able to track what entities are needed.  Same with DXCC, etc...
